docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,602 @@
++
+.Example `update_kafka_topic` module
+[source,yaml]
+----
+- name: Update Kafka Topic
+    update_kafka_topic:
+      topic_name: "kafka-topic-name"
+      kafka_id: "{{ kafka_req_resp.id }}"
+      # To hardcode the kafka_id, uncomment and use the following line:
+      # kafka_id: "KAFKA_ID"
+      partitions: 1
+      retention_period_ms: "86400000"
+      retention_size_bytes: "1073741824"
+      cleanup_policy: "delete"
+      openshift_offline_token: "OPENSHIFT_OFFLINE_TOKEN"
+    register:
+      update_topic_res_obj
+----
++
+
+. Configure the `delete_kafka_topic` module to delete both created topics.
++
+.Example `delete_kafka_topic` module
+[source,yaml]
+----
+- name: Delete Kafka Topic
+   rhoas.rhoas.delete_kafka_topic:
+     topic_name: "KAFKA_TOPIC_NAME"
+      kafka_id: "{{ kafka_req_resp_obj['kafka_id'] }}"
+      # To hardcode the kafka_id, uncomment and use the following line:
+      # kafka_id: "KAFKA_ID"
+     openshift_offline_token: "OPENSHIFT_OFFLINE_TOKEN"
+----
+
+. Configure the `delete_service_account_by_id` module to delete the service account.
++
+.Example `delete_service_account_by_id` module
+[source,yaml]
+----
+- name: Delete Service Account
+   rhoas.rhoas.delete_service_account_by_id:
+   # service_account_id: "service_account_id"
+  service_account_id: "{{ srvce_acc_resp_obj['client_id'] }}"
+
+  # openshift_offline_token: "OFFLINE_TOKEN"
+----
+
+. Deprovision and delete the {product-kafka} instance using the `delete_kafka_by_id` module.
++
+.Example `delete_kafka_by_id` module
+[source]
+----
+- name: Delete kafka instance by ID
+    rhoas.rhoas.delete_kafka_by_id:
+     kafka_id: "{{ kafka_req_resp_obj['kafka_id'] }}"
+     openshift_offline_token: "offline_token"
+----
+. Save your changes.
+. Open a terminal and enter the following command to run the example `rhoas_test` playbook:
++
+[source, shell]
+----
+$ ansible-playbook rhoas_test.yml
+----
+
+As the playbook example used in this guide is intended to test the resources, it deletes all created resources when run. Therefore, some alteration of the playbook is necessary to keep the resources. For more information on writing playbooks, see the https://docs.ansible.com/ansible/latest/user_guide/playbooks_best_practices.html#playbooks-tips-and-tricks[Ansible documentation].
+
+[id="con-terraform_{context}"]
+== Terraform
+
+link:https://www.terraform.io/[HashiCorp Terraform^] is an Infrastructure as Code tool that lets you build, change, and version infrastructure safely and efficiently through human-readable configuration files that you can version, reuse, and share. You can then use a consistent workflow to provision and manage all of your infrastructure throughout its lifecycle.
```

Comment:
A couple of comments:

- Use "enables" instead of "lets". See https://www.ibm.com/docs/en/ibm-style?topic=word-usage#l.
-Let's avoid using "version" as a verb. See https://www.ibm.com/docs/en/ibm-style?topic=word-usage#v.